docs/transactions.md

```diff
@@ -1607,19 +1532,123 @@ Gets all revoked NOC root certificates.
 - REST API:
   - GET `/dcl/pki/revoked-noc-root-certificates`
 
-### GET_REVOKED_NOC_X509_ROOT_CERT
+### Common
+
+#### GET_CERT
```

Comment:
Please clarify that it works for all types of certificates (PAA, PAI, NOC_ROOT, NOC_ICA).

Reply:
Donw